### PR TITLE
change blockr links to blockstream.info

### PR DESCRIPTION
**Pull request by jlopp**
Fixes https://github.com/GlacierProtocol/GlacierProtocol/issues/30

### CODE REVIEW

File: _docs/check-balance/overview.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ currently in one cold storage address.
 for details). If you've lost access to it, you'll need to recreate a new
 electronic copy by transcribing one of the hardcopies (stored with each private
 key) by hand.
-2. Go to [Blockr](https://www.coinbase.com/), paste your
+2. Go to [Blockstream](https://blockstream.info/), paste your
 <span class="warning">cold storage address</span> into the search bar, and
 press Enter.
 3. You'll be taken to a page that says "Bitcoin Address" at the top, with your
```
Comment (bitcoinhodler) [inline _docs/check-balance/overview.md]: Blockstream.info does not list anything called "Balance"...the number we're looking for is labeled as "Confirmed Unspent".

File: _docs/deposit/deposit-execution.md [hunk 2]
```diff
@@ -55,7 +55,7 @@ Privacy Considerations subsection for details.)
     of your funds with no recourse.</span>**
     3. Execute the transaction.
 5. Verify the deposit on the public blockchain.
-    1. Go to Blockr, paste the address into the search bar, and press Enter.
+    1. Go to Blockstream.info, paste the address into the search bar, and press Enter.
     You'll be taken to a page that says "Bitcoin Address" at the top, with your
     <span class="warning">cold storage address</span> listed underneath.
     2. Within a couple of minutes (and often much faster), you should be able
```
Comment (bitcoinhodler) [inline _docs/deposit/deposit-execution.md]: Same comment here about "Balance"...it's called "Confirmed Unspent".

File: _docs/check-balance/overview.md (hunk 1 above)
Comment (jlopp) [reply _docs/check-balance/overview.md]: updated wording

File: _docs/deposit/deposit-execution.md (hunk 2 above)
Comment (jlopp) [reply _docs/deposit/deposit-execution.md]: updated wording

File: _docs/withdrawal/execution.md [hunk 3]
```diff
@@ -30,7 +30,7 @@ yourself from your smartphone previously.
     <span class="warning">transaction ID</span> for your withdrawal; make a
     note of it.
 4. Verify the withdrawal on the public blockchain.
-    1. Go to [Blockr](https://www.coinbase.com/) , paste
+    1. Go to [Blockstream](https://blockstream.info/) , paste
     the <span class="warning">destination address</span> into the search bar,
     and press Enter. You'll be taken to a page that says "Bitcoin Address" at
     the top, with the <span class="warning">destination address</span> listed underneath.
```
Comment (bitcoinhodler) [inline _docs/withdrawal/execution.md]: This should have the same wording changes as the deposit execution ("Bitcoin Address" to "Address", "Balance" to "Confirmed Unspent")

Comment (jlopp) [reply _docs/withdrawal/execution.md]: changed to "confirmed spent" since this is for withdrawing funds, though either value would work.

Comment (bitcoinhodler) [reply _docs/withdrawal/execution.md]: We are looking at the destination address here, not the cold storage address. It should be confirmed UNspent.

Then again, the recipient might have spent it already, so this isn't an entirely reliable process.

File: _docs/withdrawal/execution.md [hunk 4]
```diff
@@ -30,13 +30,12 @@ yourself from your smartphone previously.
...
-    2. Within a couple of minutes (and often much faster), you should be able to
-    refresh this page and see your funds listed under "Unconfirmed".
-    3. Periodically refresh the page until you see the funds moved from
-    "Unconfirmed" to be reflected in "Balance". This generally happens within
-    15 minutes; if the Bitcoin network is unusually congested, it may take
-    longer.
+    2. Within a couple of seconds you should be able to refresh this page and see your
+    unconfirmed transaction at the top of the transaction list.
+    3. Periodically refresh the page until you see the funds reflected in "Confirmed Spent".
+    This may take anywhere from several minute to many hours depending upon the fee rate you
```
Comment (bitcoinhodler) [inline _docs/withdrawal/execution.md]: nit: minutes plural

**Pull request merged**